### PR TITLE
TKO-37 -- FE: Create student profile template (functionality does not really need to work yet)

### PR DESCRIPTION
**Pull request by alecnicolas**
# Description
- Created student profile view (functionality not implemented yet)
- Added navbar to the application (styling yet to be determined)  

# Testing
Navigate to `/profile` and ensure the page is responsive.

# Installation
Probably not, but `yarn install` if you run into problems.

### CODE REVIEW

File: resources/js/components/studentProfile/StudentPhoto.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+	<div class="row justify-content-end py-3 px-2">
```
Comment (andrewMETALAB) [inline resources/js/components/studentProfile/StudentPhoto.vue:2]: You don't need this div or the next one. Just move the "py-3 px-2" to this element's parent instead. 👍

File: resources/js/components/studentProfile/BadgesEarned.vue [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<template>
+	<div class="row">
```
Comment (andrewMETALAB) [inline resources/js/components/studentProfile/BadgesEarned.vue:2]: You don't need this div or the following one. Grid stuff has already been handled in the parent.

File: resources/js/components/studentProfile/CurrentInterests.vue [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<template>
+	<div class="row">
```
Comment (andrewMETALAB) [inline resources/js/components/studentProfile/CurrentInterests.vue:2]: You don't need this div or the following one. Grid stuff has already been handled in the parent.

File: resources/js/components/studentProfile/ProfileInfo.vue [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<template>
+	<div class="row">
```
Comment (andrewMETALAB) [inline resources/js/components/studentProfile/ProfileInfo.vue:2]: You don't need this div or the following one. Grid stuff has already been handled in the parent.

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
Overall great work.

(1) I left some comments throughout the code. All stuff related to the grid. (You were nesting grid columns/rows unnecessarily).

(2) Bootstrap already gives us breakpoint variables. Please remove your $break-xx variables and instead use bootstraps:  https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints

(3) Maybe instead of nesting a `<strong>` tag inside each div with a class of "profile-info__header", you could just set the ".profile-info__header" to have "font-weight:bold"

(4) Maybe current-interests doesnt need to be it's own component? I think it could just be part of the profile-info component? (It seems to me to be part of the profile-info section). 

(5) font-sizes seem way too small to me on small viewports. I don't think we should ever go below 16px for font-sizes, regardless of viewport width.

**Review by raimahossain — APPROVED** [review]
visually everything looks gooooosh!

**Review by andrewMETALAB — APPROVED** [review]
Great work 👍

**Pull request merged**